docs/synapse/devguides/devops_cell.rst

```diff
@@ -38,6 +38,10 @@ Depending on deployment requirements, a combination of methods can be used for l
     location. Any orchestration tooling should consider the requirements for cell directory data to be persistent,
     unless stated otherwise.
 
+.. warning::
+    The base Cell implements the ``mirror`` configuration option and supporting logic, but not all Synapse Cells may
+    implement their functionality in a mirroring fashion. See the specific documentation for a given Synapse Cell to
```

Comment:
maybe "mirrorable fashion" or "way that supports mirroring"  instead of "mirroring fashion"

Reply:
I'd also like to get away from talking about classes and class hierarchies in devops documentation like they should know what it means.  Lets discuss this particular little blurb :)

Reply:
Took a different approach, this is obe.